test/cpp/api/functional.cpp

```diff
@@ -929,9 +929,9 @@ TEST_F(FunctionalTest, LocalResponseNorm) {
 
 TEST_F(FunctionalTest, Linear) {
   {
-    const auto x = torch::arange(100, 118).resize_({3, 3, 2});
-    const auto w = torch::arange(200, 206).resize_({3, 2});
-    const auto b = torch::arange(300, 303);
+    const auto x = torch::arange(100., 118).resize_({3, 3, 2});
```

Comment:
why do we change this test?

Reply:
to preserve the behavior of the test, since that initializer went from returning a float tensor -> int tensor